app/models/rook.rb

```diff
@@ -1,2 +1,6 @@
 class Rook < Piece
+  def piece_rules?(to_x:, to_y:)
+    return true if to_y == y_position || to_x == x_position
```

Comment:
Yeah so you just put another line in with the x & y flipped to check for horizontal movement..
```return true if to_y > y_position && to_x == x_position || to_y < y_position && to_x == x_position``` for vertical, 
```
return true if to_x > x_position && to_y == y_position || to_x < x_position && to_y == y_position``` for horizontal. Basically holds one axis constant while checking to see if the target coordinate is higher or lower than the current position in the other axis. If that makes sense.